src/ResourceFile.cc

```diff
@@ -625,129 +560,144 @@ ResourceFile::TemplateEntryList ResourceFile::decode_TMPL(const void* data, size
   return ret;
 }
 
-static void disassemble_from_template_inner(
-    deque<string>& lines,
-    StringReader& r,
-    const ResourceFile::TemplateEntryList& entries,
-    size_t indent_level) {
-
+static string format_string(ResourceFile::TemplateEntry::Format format, const string& str, bool has_name) {

```

Comment:
Since these functions are now in the global scope, they should have more specific names, like `format_template_string_field`.